main.js

```diff
@@ -53,12 +55,16 @@ define([
              Tree) {
         "use strict";
 
+        var overrides = JSON.parse(overridesJson);
         return declare(PluginBase, {
-            toolbarName: "Regional Planning",
-            fullName: "Configure and control layers to be overlayed on the base map.",
+            toolbarName: overrides.name || "Regional Planning",
```

Comment:
These default values are redundant to the values in this original `overrides.json` file, but is a precaution against improper alteration of the overrides in regional repo sites.